### ISSUE TITLE
OS global font not followed in notifications

### ISSUE THREAD

**Issue by jay-ramani** [irzyxa/Volume2#315]
**Describe the bug**
The font set by the OS globally (via [WinAero tweaker](https://winaero.com/)) is not followed in notifications, with Volume2 using it's own font (or probably the one set by the theme). The leads to an inconsistent user interface.

There should be an option to stick to what the OS specifies.

**Desktop (please complete the following information):**
 - Windows version: Windows 11 23H2 (OS build: 22631.3085)
 - Volume² version: 1.1.9.466

Do let me know if any other information is required.

Thanks and cheers.

**Comment by irzyxa** [comment]
Yes, font should be set manually by editing skin.xml file (Font - Name or FontName attributes)

**Comment by jay-ramani** [comment]
Thanks for the update. Should the font by default not follow the used globally by the OS?
